moped-etl/prefect/moped_test_create_flow.py

```diff
@@ -96,7 +96,7 @@ def drain_service(slug):
 
 
 # with Flow("Moped Test Instance Commission") as test_commission:
-with Flow("Moped Test Instance Commission", executor=executor) as test_commission:
+with Flow("Moped Test Instance Commission") as test_commission:
```

Comment:
this was done in the code that's on atd-data03, but hasnt be pushed/merged to the code in github